core/shared/src/main/scala/org/http4s/EntityEncoder.scala

```diff
@@ -231,3 +229,16 @@ object EntityEncoder {
       .contramap[EventStream[F]](_.through(ServerSentEvent.encoder))
       .withContentType(`Content-Type`(MediaType.`text/event-stream`))
 }
+
+trait Utf8EntityEncoders {
+  implicit val utf8StringEncoder: EntityEncoder.Pure[String] =
+    EntityEncoder.stringEncoder(Charset.`UTF-8`)
+
+  implicit val utf8CharArrayEncoder: EntityEncoder.Pure[Array[Char]] =
+    utf8StringEncoder.contramap(new String(_))
+
+  implicit def utf8ReaderEncoder[F[_]: Sync, R <: Reader]: EntityEncoder[F, F[R]] =
+    EntityEncoder.readerEncoder[F, R](Charset.`UTF-8`)
+}
+
+object Utf8EntityEncoders
```

Comment:
What is the reasoning to keep these separate?

Reply:
Hmm, I haven't a strong opinion of this. Just this seemed to be more modular for me.

Reply:
Do you have an idea of how it would be used? If it's just for keeping our sources tidy then I would prefer we keep `Utf8EntityEncoders` package-private to avoid expanding our API surface unnecessarily.